run.go

```diff
@@ -67,20 +69,49 @@ func run(context *cli.Context) error {
 		return errors.New("invalid runtime config")
 	}
 
-	if err := create(context.Args().First(),
+	consolePath := context.String("console")
+
+	var console Console
+	var err error
+	var consoleState *term.State
+
+	/* if consolePath is /dev/ptmx means that we have to allocate a new pts */
```

Comment:
/dev/ptmx or /dev/pts/ptmx, actually old-ish runc is uses /dev/pts/ptmx in its tests.